### ISSUE TITLE
Cannot use child selector with stylex

### ISSUE THREAD

**Issue by phamthanhtrungg** [ladifire-opensource/stylex#15]
<img width="381" alt="Screen Shot 2021-02-16 at 00 00 41" src="https://user-images.githubusercontent.com/31823932/107975081-20e76c80-6fea-11eb-949c-dcbae27f6548.png">
<img width="551" alt="Screen Shot 2021-02-16 at 00 03 35" src="https://user-images.githubusercontent.com/31823932/107975246-6b68e900-6fea-11eb-998b-e59adef0eede.png">

**Comment by ladifire** [comment]
@phamthanhtrungg
yes, child selector is not supported yet. I've marked it as a feature for next version.
